near-sdk-macros/src/core_impl/abi/abi_generator.rs

```diff
@@ -329,7 +333,7 @@ mod tests {
                 callbacks: vec![],
                 callbacks_vec: None,
                 result: Some(near_sdk::__private::AbiType::Json {
-                    type_schema: gen.subschema_for::<IsOk>(),
+                    type_schema: gen.subschema_for::< <Result<IsOk, Error> as near_sdk::__private::ResultTypeExt>::Okay>(),
```

Comment:
Just an informational comment: I had trouble eyeballing whether this is syntactically correct so I ran the compilation tests with feature `__abi-generate` and they passed 👍